docs/docs/install/docker-compose.md

```diff
@@ -0,0 +1,22 @@
+# Run Barco with Docker Compose
```

Comment:
I've noticed that files in `docs/` have been copied to `docs/docs/`, is that intended?

Would it be possible to avoid this duplication and leave files under `docs/`?
If the existing md files need to be renamed, e.g. to lower case, we could use `git mv`.

Reply:
Sorry for the delay...
Yes, it's possible to have the docs directly in the docs folder, in that case, the mkdocs.yml should be in the project root folder.
Regarding the file names, it is not necessary to rename them, I will keep the names in capital letters.
Do I move forward with it?

Reply:
finallly push a refactor. Let me know if something is wrong